src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx

```diff
@@ -19,13 +20,77 @@ type HacbsApplicationDetailsProps = {
+          kind: 'QuickStarts',
+          metadata: {
+            name: 'hacbs-gs-apps',
+          },
+          spec: {
+            version: 4.7,
+            displayName: `Getting started with an app`,
+            durationMinutes: 60,
+            icon: 'data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBHZW5lcmF0b3I6IEFkb2JlIElsbHVzdHJhdG9yIDI1LjIuMCwgU1ZHIEV4cG9ydCBQbHVnLUluIC4gU1ZHIFZlcnNpb246IDYuMDAgQnVpbGQgMCkgIC0tPgo8c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IgoJIHZpZXdCb3g9IjAgMCAzNyAzNyIgc3R5bGU9ImVuYWJsZS1iYWNrZ3JvdW5kOm5ldyAwIDAgMzcgMzc7IiB4bWw6c3BhY2U9InByZXNlcnZlIj4KPHN0eWxlIHR5cGU9InRleHQvY3NzIj4KCS5zdDB7ZmlsbDojRUUwMDAwO30KCS5zdDF7ZmlsbDojRkZGRkZGO30KPC9zdHlsZT4KPGc+Cgk8cGF0aCBkPSJNMjcuNSwwLjVoLTE4Yy00Ljk3LDAtOSw0LjAzLTksOXYxOGMwLDQuOTcsNC4wMyw5LDksOWgxOGM0Ljk3LDAsOS00LjAzLDktOXYtMThDMzYuNSw0LjUzLDMyLjQ3LDAuNSwyNy41LDAuNUwyNy41LDAuNXoiCgkJLz4KCTxwYXRoIGNsYXNzPSJzdDEiIGQ9Ik0yNS41MSwyMi4xN0gxMi42NGMtMi45OCwwLTUuNDEtMi40Mi01LjQxLTUuNDFjMC0zLjE1LDIuNjctNS43LDUuOTItNS4zOGMxLjE4LTIuMjcsMy41Mi0zLjcsNi4xLTMuNwoJCWMzLjc1LDAsNi44MSwzLjAyLDYuODgsNi43NmMxLjcxLDAuNDQsMi45NCwxLjk4LDIuOTQsMy43OWMwLDEuOTYtMS40NSwzLjYtMy4zMywzLjg5QzI1LjY4LDIyLjE1LDI1LjYsMjIuMTcsMjUuNTEsMjIuMTd6CgkJIE0xMi42NCwxMi42MWMtMi4yOSwwLTQuMTYsMS44Ni00LjE2LDQuMTVzMS44Niw0LjE2LDQuMTYsNC4xNmgxMi41YzEuNDgsMCwyLjY4LTEuMiwyLjY4LTIuNjhjMC0xLjM3LTEuMDMtMi41MS0yLjQtMi42NQoJCWMtMC4zMy0wLjA0LTAuNTgtMC4zMi0wLjU2LTAuNjZsMC4wMS0wLjEyYzAuMDEtMC4wOCwwLjAxLTAuMTcsMC4wMS0wLjI1YzAtMy4xLTIuNTItNS42My01LjYzLTUuNjNjLTIuMjQsMC00LjI3LDEuMzMtNS4xNiwzLjM5CgkJYy0wLjEyLDAuMjctMC40LDAuNDItMC42OSwwLjM3QzEzLjE1LDEyLjYzLDEyLjg5LDEyLjYxLDEyLjY0LDEyLjYxeiIvPgoJPHBhdGggY2xhc3M9InN0MCIgZD0iTTI1LjUsMjMuODdjLTEuMjMsMC0yLjI2LDAuODUtMi41NCwyaC0xLjkyYy0wLjIzLTAuOTQtMC45Ny0xLjY4LTEuOTItMS45MnYtNy45NWwwLjkzLDAuOTMKCQljMC4xMiwwLjEyLDAuMjgsMC4xOCwwLjQ0LDAuMThzMC4zMi0wLjA2LDAuNDQtMC4xOGMwLjI0LTAuMjQsMC4yNC0wLjY0LDAtMC44OGwtMi0yYy0wLjI0LTAuMjQtMC42NC0wLjI0LTAuODgsMGwtMiwyCgkJYy0wLjI0LDAuMjQtMC4yNCwwLjY0LDAsMC44OGMwLjI0LDAuMjQsMC42NCwwLjI0LDAuODgsMGwwLjkzLTAuOTN2Ny45NWMtMC45NCwwLjIzLTEuNjgsMC45Ny0xLjkyLDEuOTJoLTEuOTIKCQljLTAuMjgtMS4xNS0xLjMxLTItMi41NC0yYy0xLjQ1LDAtMi42MiwxLjE4LTIuNjIsMi42MnMxLjE4LDIuNjIsMi42MiwyLjYyYzEuMjMsMCwyLjI2LTAuODUsMi41NC0yaDEuOTJjMC4yOCwxLjE1LDEuMzEsMiwyLjU0LDIKCQlzMi4yNi0wLjg1LDIuNTQtMmgxLjkyYzAuMjgsMS4xNSwxLjMxLDIsMi41NCwyYzEuNDUsMCwyLjYyLTEuMTgsMi42Mi0yLjYyUzI2Ljk1LDIzLjg3LDI1LjUsMjMuODd6IE0xMS41LDI3Ljg3CgkJYy0wLjc2LDAtMS4zOC0wLjYyLTEuMzgtMS4zOHMwLjYyLTEuMzgsMS4zOC0xLjM4czEuMzgsMC42MiwxLjM4LDEuMzhTMTIuMjYsMjcuODcsMTEuNSwyNy44N3ogTTE4LjUsMjcuODcKCQljLTAuNzYsMC0xLjM4LTAuNjItMS4zOC0xLjM4czAuNjItMS4zOCwxLjM4LTEuMzhzMS4zOCwwLjYyLDEuMzgsMS4zOFMxOS4yNiwyNy44NywxOC41LDI3Ljg3eiBNMjUuNSwyNy44NwoJCWMtMC43NiwwLTEuMzgtMC42Mi0xLjM4LTEuMzhzMC42Mi0xLjM4LDEuMzgtMS4zOHMxLjM4LDAuNjIsMS4zOCwxLjM4UzI2LjI2LDI3Ljg3LDI1LjUsMjcuODd6Ii8+CjwvZz4KPC9zdmc+Cg==',
```

Comment:
What is this icon?

Reply:
it's not required so i remove it in the latest version